bootstrap-sass/src/main/java/de/agilecoders/wicket/sass/UrlImporter.java

```diff
@@ -231,8 +232,8 @@ private String addUnderscore(String url) {
 
     private String getAbsolutePath(URI base, String url) {
         String basePath = base.toString();
-        Path parentBasePath = Paths.get(basePath).getParent();
```

Comment:
IMO this should be `Path.of(base)`, i.e. use the `URI`, instead of `String`.
Could you please test it on Windows ?

Reply:
If you do that, then you get `Method threw 'java.nio.file.FileSystemNotFoundException' exception.`

Reply:
That would be awesome!